app/src/main/kotlin/at/bitfire/davdroid/ui/widget/CardWithImage.kt

```diff
@@ -0,0 +1,78 @@
+/***************************************************************************************************
+ * Copyright © All Contributors. See LICENSE and AUTHORS in the root directory for details.
+ **************************************************************************************************/
+
+package at.bitfire.davdroid.ui.widget
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import at.bitfire.davdroid.R
+
+@Composable
+fun CardWithImage(
+    image: Painter,
+    title: String,
+    message: String,
+    modifier: Modifier = Modifier,
+    contentDescription: String? = null,
+    content: @Composable ColumnScope.() -> Unit
```

Comment:
We can set it to `{}` by default, then we don't need to specify it in the preview. And maybe we want to show a card somewhere that has only a title and a message (without content).